From 2211b4c9c27d6ec566dc95b0361ed3e7b4b820e6 Mon Sep 17 00:00:00 2001
From: Philip Balister <philip@balister.org>
Date: Mon, 16 Dec 2013 15:40:24 -0500
Subject: [PATCH 24/25] gr-fft-filter : Fix uninitialized pointers.

Address CID's 1043252 and 1043253. Inspection suggest the code should
work, but set the pointers to NULL to force errors on strange code
paths.

Signed-off-by: Philip Balister <philip@balister.org>
---
 gr-filter/lib/fft_filter.cc |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gr-filter/lib/fft_filter.cc b/gr-filter/lib/fft_filter.cc
index b4cea0e..8293757 100644
--- a/gr-filter/lib/fft_filter.cc
+++ b/gr-filter/lib/fft_filter.cc
@@ -39,7 +39,7 @@ namespace gr {
 				     const std::vector<float> &taps,
 				     int nthreads)
 	: d_fftsize(-1), d_decimation(decimation), d_fwdfft(0),
-	  d_invfft(0), d_nthreads(nthreads)
+	  d_invfft(0), d_nthreads(nthreads), d_xformed_taps(NULL)
       {
 	set_taps(taps);
       }
@@ -192,7 +192,7 @@ namespace gr {
 				     const std::vector<gr_complex> &taps,
 				     int nthreads)
 	: d_fftsize(-1), d_decimation(decimation), d_fwdfft(0),
-	  d_invfft(0), d_nthreads(nthreads)
+	  d_invfft(0), d_nthreads(nthreads), d_xformed_taps(NULL)
       {
 	set_taps(taps);
       }
-- 
1.7.10.4

